lib/codes/itunes_connect.rb

```diff
@@ -40,6 +40,16 @@ def run(args)
 
       text_fields.first.set(number_of_codes.to_s)
       click_next
+
+      # are there any errors ?
+      errors = []
+      begin
+         errors = wait_for_elements("div[id=LCPurpleSoftwarePageWrapperErrorMessage]")
+      rescue Exception => e
+         # puts "Error: #{e.message}"
```

Comment:
On Sep 2, 2015 19:42, "Felix Krause" <notifications@github.com> wrote:
>
> In lib/codes/itunes_connect.rb:
>
> > @@ -40,6 +40,16 @@ def run(args)
> >
> >        text_fields.first.set(number_of_codes.to_s)
> >        click_next
> > +
> > +      # are there any errors ?
> > +      errors = []
> > +      begin
> > +         errors =
wait_for_elements("div[id=LCPurpleSoftwarePageWrapperErrorMessage]")
> > +      rescue Exception => e
> > +         # puts "Error: #{e.message}"
>
> I think we don't want to print the error message here, right? We should
remove the comment then.

I've noticed that there's an error message printed when it fails. Maybe
coming from wait_for_elementos.

I have to check it again.